app/src/main/java/dizel/budget_control/budget/view/create_budget/CreateBudgetFragment.kt

```diff
@@ -76,11 +101,19 @@ class CreateBudgetFragment: Fragment(R.layout.fragment_create_budget) {
         }
     }
 
-    private fun showError(mes: String) {
-        AlertDialog.Builder(requireContext())
-            .setTitle(R.string.error_stub_title)
-            .setMessage(mes)
-            .setPositiveButton("OK") { dialog, _ -> dialog.dismiss() }
-            .show()
+    @Composable
+    fun ErrorDialog(message: String, openDialog: MutableState<Boolean>) {
```

Comment:
Replace MutableState<Boolean> to just Boolean

Reply:
Replace MutableState<Boolean> to MutableState<Integer>